### ISSUE TITLE
The plugin breaks partial classes like `text-{{ i }}` in Twig files

### ISSUE THREAD

**Issue by alessandro-newzoo** [tailwindlabs/prettier-plugin-tailwindcss#183]
## **What version of `prettier-plugin-tailwindcss` are you using?**

`0.4.0`

## **What version of Tailwind CSS are you using?**

`3.3.3`

## **What version of Node.js are you using?**

`20.4.0`

## **What package manager are you using?**

`npm`

## **What operating system are you using?**

`Linux codespaces-4330ea 5.15.0-1041-azure #48-Ubuntu SMP`

## **Reproduction**

```twig
<div class="flex text-{{ i }}">
    xxx
</div>
```

will become:

```twig
<div class="text- flex{{ i }}">
    xxx
</div>
```

## `.prettierrc.js`:
```js
module.exports = {

	plugins: [
		'./node_modules/prettier-plugin-twig-melody',
		require('prettier-plugin-tailwindcss')
	],
}

```

## **Describe your issue**

Hello!
First of all thank you so much for all your work on this fantastic plugin 🙏🏻

I was excited to see this issue I've been having was finally fixed in #164 , though it's not working for me, with .twig files.
I'm not sure if it's because the fix is specific to Liquid files, or because I'm setting something wrong up, but either way I thought I'd report it and kindly ask if it's possible to make this work with Twig files too?

Happy Friday!
Alessandro

**Comment by thecrypticace** [comment]
Hey! Each language is generally quite different and just didn't think / realize this needed to be handled. I've merged the fix for this in #184 and have tagged v0.4.1 that includes it.

Let me know if you notice anything else going on. ✨

**Comment by alessandro-newzoo** [comment]
hi @thecrypticace you're an absolute legend, thank you so much for such a prompt fix!
You have no idea for how long I've been struggling with this problem and how annoying it was. My life is gonna be so much better now, thank you ❤️
